torch/onnx/symbolic_opset9.py

```diff
@@ -6302,6 +6302,22 @@ def is_pinned(g: jit_utils.GraphContext, self, device=None):
     return None
 
 
+@_onnx_symbolic("aten::resolve_conj")
+def resolve_conj(g: jit_utils.GraphContext, self: _C.Value) -> _C.Value:
```

Comment:
`def _conj` is needed too. By setting `GLABLS.onnx_shape_inference=False`, the exporter would advance a little further and complain about the missing `aten::_conj`. See https://github.com/pytorch/pytorch/pull/86223 which has all 3 symbolics, plus some type checking to make sure input is never complex types

Reply:
Cool, I'll close this one and focus on your version. Will be pasting discussion and comments there.